vlm/src/main/scala/geotrellis/contrib/vlm/RasterSource.scala

```diff
@@ -42,7 +40,7 @@ import java.io.Closeable
   * @groupdesc reproject Functions to resample raster data in target projection.
   * @groupprio reproject 2
   */
-trait RasterSource extends CellGrid with Closeable with Serializable {
+trait RasterSource extends CellGrid with AutoCloseable with Serializable {
```

Comment:
Whats the practical difference between `Closable` and `AutoClosable` why was this switch helpful ?

Reply:
So the `Closeable` is a legacy type with a more restricted exception type (it can throw only `IOException`). `AutoCloseable` can be used with Java >= 1.7 only.